byte.h

```diff
@@ -7,8 +7,8 @@ extern unsigned int byte_chr();
 extern unsigned int byte_rchr();
 extern void byte_copy();
 extern void byte_copyr();
-extern void byte_zero();
 
+#define byte_zero(s,n) (memset((s),0,(n)))
```

Comment:
In-tree, `byte_zero()` is called from three places:

1. `qmail-popup.c`, with the intent of clearing username/password/timestamp from memory ASAP
2. `timeoutconn.c` and `remoteinfo.c`, to zero out a new struct

For (2), I'm less worried about what compilers will optimize out; this looks like an idiomatic use of `memset()`.

For (1), I'm more worried. How can we tell whether the existing hand-rolled `byte_zero()` implementation has the desired effect, and whether a `memset()` preserves the expected behavior?

Out of tree, I know of at least my own `authup` program that's derived from `qmail-popup` and has the same expectations for `byte_zero()`.